src/data/data.js

```diff
@@ -58,20 +58,21 @@ const pageData = {
       {
         component: 'scenarios',
         title: 'Escenarios que utilizamos',
-        heading: 'Piscina de actividades Sub-acuáticas',
-        text: 'CDPD',
         images: [
           {
-            src: '',
-            alt: 'Piscina',
+            src: 'basketball.png',
+            alt: 'Basketball',
+            text: 'CDPD',
           },
           {
-            src: '',
+            src: 'piscinaescuela.png',
```

Comment:
and this image has the text inside the image, which is part of the image, that is why the styles, do not affect this text. is an error with the image.

Reply:
Done. I changed the name of the images with the convention they should have. And yes your right it's a design issue.